modules/apps/foundation/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/util/ConfigurationModelIterator.java

```diff
@@ -44,7 +45,22 @@ public ConfigurationModelIterator(
 	}
 
 	public List<ConfigurationModel> getResults(int start, int end) {
-		return ListUtil.subList(_configurationModels, start, end);
+		return getResults(start, end, null);
+	}
+
+	public List<ConfigurationModel> getResults(
+		int start, int end, Comparator<ConfigurationModel> comparator) {
+
+		if (comparator == null) {
+			return ListUtil.subList(_configurationModels, start, end);
```

Comment:
This is a bit weird. If there is no shared code with getResults(int start, int end) {} then don't make it invoke this method. Instead this method should invoke getResults(int start, int end)  when comparator is null.